packages/workbox-sw/index.mjs

```diff
@@ -24,6 +24,14 @@ import './_version.mjs';
  */
 
 const CDN_PATH = `WORKBOX_CDN_ROOT_URL`;
+const MODULE_KEY_TO_NAME_MAPPING = {
+  'expiration': 'workbox-cache-expiration',
+  'strategies': 'workbox-runtime-caching',
+};
+const MODULE_NAME_TO_KEY_MAPPING = {
```

Comment:
It might be nice to maintain one copy of this mapping, and then automate the reversal:

```js
const MODULE_NAME_TO_KEY_MAPPING = {};
for (const key of Object.keys(MODULE_KEY_TO_NAME_MAPPING)) {
  const name = MODULE_KEY_TO_NAME_MAPPING[key];
  MODULE_NAME_TO_KEY_MAPPING[name] = key;
}
```

And then optionally you could run a sanity check to make sure there's really a 1:1 mapping:

```js
if (process.env.NODE_ENV !== 'production') {
  if (Objects.keys(MODULE_KEY_TO_NAME_MAPPING).length !== Object.keys(MODULE_NAME_TO_KEY_MAPPING).length) {
    throw WorkboxError();
  }
}
```

Reply:
I like the idea of minimizing copies of the mapping here. +1 to Jeff's suggestion.

Reply:
I've automated the switch around. Would be good to just automate the creation of these altogether so it's in-sync with the package.json namespaces.